jobs/furnishings/src/furnishings/config.py

```diff
@@ -125,6 +143,12 @@ class TestConfig(_Config):  # pylint: disable=too-few-public-methods
         name=DB_NAME,
     )
 
+    # BCLaws SFTP
+    BCLAWS_SFTP_STORAGE_DIRECTORY = 'bclaws'
+
+    # BCMail+ SFTP
+    BCMAIL_SFTP_STORAGE_DIRECTORY = 'bcmail'
```

Comment:
It's better to get these 2 directories from `.env` because we need to set up them later.

Reply:
These directories in `TestConfig` are only used in the unit tests, and never actually connect to an SFTP server, so we don't need to set these directories up at all. The unit tests use `pytest-sftpserver`, which is an in-memory sftp "server" for pytests. Since it requires no setup and is not used outside of the unit tests I still think it might be better to keep these values hard-coded (like how `legal-api` has some hard-coded values in `TestConfig`).

There are also a couple quirks with using `pytest-sftpserver` that I go into more detail in [this comment](https://github.com/bcgov/lear/pull/2964#discussion_r1750624263).

Reply:
You're correct and thanks for explanation! 😄 I mistook these in test config for the ones in base config.